### DISCUSSION TITLE
Issue regarding Repository Settings

### DISCUSSION THREAD

**Discussion by Unknwnguy** [community/community#33345] [Other Feature Feedback, Questions, & Ideas]
I have recently opened my GitHub account. In my repository, the settings options is missing. I have searched some websites about this issue but, i didn't get any solutions regarding this issue. If anyone knows about this issue, Kindly give some solution.

**Comment by queenofcorgis** [comment]
Hi there! Can you elaborate and include screenshots if possible so we can better assist please? Thanks 🤩

**Reply by ronwest** [reply]
![missingSettings](https://user-images.githubusercontent.com/448748/216111837-f03ced0c-1574-4630-ad73-31c832440add.PNG)
I am having the same issue. The settings tab is missing from where the [instructions ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository)indicates where it would be in order to add a collaborator to a private repository.

**Comment by Anmol-Baranwal** [comment]
You would not have enough permissions to access the repo settings if the repo belongs to an 'organization'.

There could be other ways to solve this -

- Maybe you are not signed in GitHub, you just have to login.
- Clearing the cache and cookies is the nuclear option, it may work (try this)
- It maybe just limited to one browser, check if the same issue occurs in different browser e.g. Microsoft Edge, Mozilla Firefox, Chrome

**Reply by ronwest** [reply]
1) I am obviously signed into github.
2) Clearing cache did not work. Neither did using incognito mode.
3) This issue is not connected to the browser.

The repository in question is mine. The original coder, who wrote and managed it under his personal github account, passed away in early 2022. I was able to gain ownership over the repository via GitHub Recovery. However, it appears that I do not have full access. Who do I need to contact to remedy this issue?

**Reply by Anmol-Baranwal** [reply]
You can `Transfer ownership` from that profile to your personal GitHub profile, that would certainly give you the required access.

**Reply by Anmol-Baranwal** [reply]
@Unknwnguy 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
